templates/detailItems/adr.html

```diff
@@ -2,10 +2,6 @@
 	<option ng-repeat="option in ctrl.availableOptions" value="{{option.id}}">{{option.name}}</option>
 </select>
 <div>
-	<label for="details-pobox">{{ctrl.t.poBox}}</label>
```

Comment:
@jancborchardt please leave the po box in here - it's a required part of the address

https://tools.ietf.org/html/rfc6350#section-6.3.1